python-template/{{cookiecutter.placeholder_repo_name}}/setup.py

```diff
@@ -127,9 +127,7 @@ def package_data(pkg, roots):
     author='edX',
     author_email='oscm@edx.org',
     url='https://github.com/edx/{{ cookiecutter.repo_name }}',
-    packages=[
-        '{{ cookiecutter.sub_dir_name }}',
-    ],
+    packages=find_packages(exclude=["test_utils", "*tests"]),
```

Comment:
This should be fine. But if we want to follow the spirit of the initial implementation, we would only include the submodules of the plugin subdirectory:

```
find_packages(include=["{{ cookiecutter.repo_name }}*"], exclude=["test_utils", "*tests"])
```

Reply:
Sounds good. I will also remove `test_utils`, since the purpose of that was to exclude the top-level test_utils directory.